### ISSUE TITLE
colspan

### ISSUE THREAD

**Issue by zine56** [JLynch7/SlickGrid#65]
hello, i was wondering if this fork allows colspan, pls mr. lynch heelp

**Comment by JLynch7** [comment]
Yes, colspan is supported.  If you have any issues, please let me know.

**Comment by zine56** [comment]
i'm trying to freeze the first row and making the duration column with a colspan = 2, but this is the result, the duration column appears blank :(

this is the result:

![image](https://f.cloud.github.com/assets/792050/1955819/275453f4-81ee-11e3-920e-11f8bc85dc6f.png)

this is the code, it seems pretty straightforward to me :(, can you help me pls?

https://gist.github.com/zine56/8523090
